dashboard/config/routes.rb

```diff
@@ -345,6 +345,7 @@ module OPS
     get 'workshop_dashboard', to: 'workshop_dashboard#index'
 
     get 'teacher_application', to: 'teacher_application#new'
+    get 'teacher_application/international_teachers', to: 'teacher_application#international_teachers'
```

Comment:
just to be clear, nothing points to this route yet, but this will be redirected by the backend when a country other than US is selected (in a subsequent PR)?